apps/challenges/views.py

```diff
@@ -248,6 +249,30 @@ def challenge_detail(request, challenge_host_team_pk, challenge_pk):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+@api_view(['GET'])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def participant_team_detail_for_challenge(request, challenge_pk):
+    """
+    API to return the participated team name in the challenge.
```

Comment:
can you also add 1 liners for each arguments in the docstrings? See this for an example https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/utils.py#L313